src/CalcViewModel/Common/KeyboardShortcutManager.cpp

```diff
@@ -579,9 +580,9 @@ void KeyboardShortcutManager::OnKeyDownHandler(CoreWindow^ sender, KeyEventArgs^
             // Writer lock for the static maps
             reader_writer_lock::scoped_lock lock(s_keyboardShortcutMapLock);
 
-            auto currentControlKeyPressed = s_ControlKeyPressed.find(viewId);
```

Comment:
What was the warning that desired this variable name change?

Reply:
The variable name was already declared earlier in the function, and declaring the same name in the nested block hides the variable from earlier in the function.